{"name": "Невозможно авторизовать пользователя, если хотя бы одно из полей неверно", "status": "passed", "parameters": [{"name": "field", "value": "'email'"}], "start": 1719747479633, "stop": 1719747480008, "uuid": "476c9c4d-a4f3-437e-a3d2-6ab56b3c217e", "historyId": "b8eb9af926b8187d9f932fbbb6fba126", "testCaseId": "75e5e0134f47bb9cc437e9df45383da3", "fullName": "tests.test_user.TestUserLogin#test_user_login_with_wrong_data_failed", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_user"}, {"name": "subSuite", "value": "TestUserLogin"}, {"name": "host", "value": "thinkpad"}, {"name": "thread", "value": "26668-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_user"}]}